### PR TITLE
New Tutorials!

### PR DESCRIPTION
**Pull request by royendo**
A big one,

Introducing Tutorials! 
This will create a new tab in the header for our docs (some reorganizing here) for a tutorial based section.

The topics that are here are part of a new github example project that will be added when this tutorial is released.
The main topics are 
- Rill Developer to Rill Cloud in 5 steps!
- Rill Advanced Features
- Adminsitration Topics
- Rill and ClickHouse to Dashboarding in 4 steps!
- Other topics

The main idea is that for "most" of these tutorials, they will have a 1to1 to a file in the github sample project. I decided to remove some of the Rill Cloud features as we should just have these better documented in the `docs` section.

Table of Contents:
Screenshot 2024-08-28 at 11 42 52

### CODE REVIEW

File: docs/docs/tutorials/rill_advanced_features/custom_dashboards/template-charts.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  
+    ```
+  </TabItem>
+</Tabs>
+
+
+Additional Templates available soon : 
+
+- Pie Chart
+- Table
```
Comment (mindspank) [inline docs/docs/tutorials/rill_advanced_features/custom_dashboards/template-charts.md:84]: Will be available on release

File: docs/docs/tutorials/rill_advanced_features/custom_dashboards/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+description: Creating Custom Dashboards in Rill
+sidebar_label: "Getting Started"
+sidebar_position: 6
+---
+
+## Creating Custom Dashboards in Rill 
+
+In this section, we'll cover how to create custom dashboards in Rill Developer and publish these to Rill Cloud,
+
+:::note
```
Comment (mindspank) [inline docs/docs/tutorials/rill_advanced_features/custom_dashboards/getting-started.md:12]: Should go live in 0.50 so we could probably omit this section if we intend the tutorials to go out at the same time

File: docs/docs/tutorials/rill_advanced_features/custom_dashboards/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+```
+rill start 
+```
+
+### Step 2: Adding feature flag (no longer required once GA)
+
+Edit the rill.yaml via the UI and add the following:
+```
+features:
```
Comment (mindspank) [inline docs/docs/tutorials/rill_advanced_features/custom_dashboards/getting-started.md:43]: See above comment re release timing.

File: docs/docs/tutorials/rill_advanced_features/custom_dashboards/template-charts.md (hunk 1 above)
Comment (royendo) [reply docs/docs/tutorials/rill_advanced_features/custom_dashboards/template-charts.md:84]: sweet!

**Review by mindspank — APPROVED** [review]

**Comment by royendo** [comment]
@mindspank How do you prefer having all the tutorials sub menus also open like the docs are, or have those selectable (as it is now)?

**Pull request merged**